packages/main/src/TableRow.js

```diff
@@ -28,6 +29,26 @@ const metadata = {
 		},
 	},
 	properties: /** @lends sap.ui.webcomponents.main.TableRow.prototype */ {
+		/**
+		 * Defines the mode of the row (None, SingleSelect, MultiSelect).
+		 * @type {TableMode}
+		 * @defaultvalue "None"
+		 * @private
+		 */
+		mode: {
+			type: TableMode,
+			defaultValue: TableMode.None,
+		},
+		/**
+		 * Defines the row's selected state.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
```

Comment:
We might need to make this property public, otherwise the component users  can't set initial state.

Reply:
Done